src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -222,6 +224,28 @@ public ResponseEntity<Object> getValidationStatus(
         return ResponseEntity.notFound().build();
     }
 
+    @GetMapping("/{overseas_entity_id}")
+    public ResponseEntity<Object> getSubmission(
+            @RequestAttribute(TRANSACTION_KEY) Transaction transaction,
+            @PathVariable(OVERSEAS_ENTITY_ID_KEY) String submissionId,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) {
+
+        var logMap = new HashMap<String, Object>();
+        logMap.put(OVERSEAS_ENTITY_ID_KEY, submissionId);
+        logMap.put(TRANSACTION_ID_KEY, transaction.getId());
+
+        try {
+            ApiLogger.infoContext(requestId, "Calling service to get the overseas entity submission", logMap);
+            return overseasEntitiesService.getSavedOverseasEntity(transaction, submissionId, requestId);
+        } catch (SubmissionNotFoundException e) {
+            ApiLogger.errorContext(requestId, e.getMessage(), e, logMap);
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        } catch (Exception e) {
+            ApiLogger.errorContext(requestId, "Error Updating Overseas Entity Submission", e, logMap);
```

Comment:
error message should say 'getting' instead of 'updating'

Reply:
This has been changed.